pkgs/test_core/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.5.2
+
```

Comment:
need an entry here?

Reply:
Yeah, probably shouldn't leave it blank...